docs/instruments/NDB_BVL_Instrument_TEMPLATE.class.inc

```diff
@@ -1,89 +1,130 @@
-    * @access public
-    */
-    function setup($commentID = NULL, $page = NULL){
-        $this->formType="XIN";
-        $this->form = new LorisForm('test_form');
-        $this->page = $page;            // page label (number or
+    function setup($commentID, $page)
+    {
+        $this->formType ="XIN";
+        $this->form     = new LorisForm('test_form');
+        $this->page     = $page;            // page label (number or
         // string - used by
         // user-defined child classes)
-        
+
         // set the object properties
-        $this->testName = "<TEST_NAME>";           // test_names.Test_name
-        $this->table = '<TEST_NAME>';              // name of database table corresponding to instrument
+        // Corresponds to the Test_name column in test_names table
+        $this->testName = "<TEST_NAME>";
+        // name of database table corresponding to instrument
+        $this->table = '<TEST_NAME>';
         // data keyed by commentID
         $this->commentID = $commentID;
-        
+
         //The array of dates/timestamps to convert to database dates/timestamps
         //Any HTML_Quickform date elements must be listed here
-        $this->dateTimeFields=array("Date_taken");
-            
+        $this->dateTimeFields =array("Date_taken");
+
         //The array of selects with multiple answers allowed
         //Any HTML_Quickform multiple selects must be listed here
         $this->_selectMultipleElements = array();
 
         // required fields for data entry completion status
-        $this->_requiredElements = array('Examiner', '<FIRST QUESTION OF EACH PAGE>');
+        $this->_requiredElements = array(
+                                    'Examiner',
+                                    '<FIRST QUESTION OF EACH PAGE>',
+                                   );
 
         // setup the form
         $this->_setupForm();
 
     }
-    
-    //If the instrument is not paged, remove the switch from the _setupForm method and add all the form Elements in this function.
-    
+
     /**
-    * method to build the HTML_Quickform object into a paged form
-    *
-    * @return void
-    * @access private
-    */
-    function _setupForm(){
-       if(preg_match("/<TEST_NAME>(_page[0-9]+)/",$this->page,$matches)){
-            call_user_func(array($this, $matches[1]));
+     * Builds the HTML_Quickform object into a paged form
+     *
+     * @return void
+     * @access private
+     */
+    function _setupForm()
+    {
+        if (preg_match("/<TEST_NAME>(_page[0-9]+)/", $this->page, $matches)) {
+            $this->_page($matches[1]);
         } else {
             $this->_main();
         }
-        //Defines the call back function for HTML Quickform to use when validating the form.
-        $this->form->addFormRule(array(&$this,'XINValidate'));
+        // Defines the call back function for HTML Quickform to use in validation
+        $this->form->addFormRule(array(&$this, 'XINValidate'));
```

Comment:
Shouldn't it be `([&$this, 'XINValidate'])` instead of `array(...)`?

Reply:
No our standard for declaring arrays in PHp is using `array()` instead of `[]`.